src/components/TableBodyContainer.js

```diff
@@ -5,31 +5,33 @@ import compose from 'recompose/compose';
 import mapProps from 'recompose/mapProps';
 import getContext from 'recompose/getContext';
 
-import { visibleRowIdsSelector, classNamesForComponentSelector, stylesForComponentSelector } from '../selectors/dataSelectors';
+//import { visibleRowIdsSelector, classNamesForComponentSelector, stylesForComponentSelector } from '../selectors/dataSelectors';
 
 const ComposedTableBodyContainer = OriginalComponent => compose(
   getContext({
     components: PropTypes.object,
     selectors: PropTypes.object,
   }),
-  connect((state, props) => ({
-    visibleRowIds: visibleRowIdsSelector(state),
-    className: classNamesForComponentSelector(state, 'TableBody'),
-    style: stylesForComponentSelector(state, 'TableBody'),
-  })),
+  connect(
+    (state, props) => ({
+      visibleRowIds: props.selectors.visibleRowIdsSelector(state),
+      className: props.selectors.classNamesForComponentSelector(state, 'TableBody'),
+      style: props.selectors.stylesForComponentSelector(state, 'TableBody'),
+    })
+  ),
   mapProps(props => {
-    const { components, ...otherProps } = props;
+    const { components, selectors, ...otherProps } = props;
```

Comment:
I noticed this is the only place you're pulling `selectors` out of `otherProps`. I think it's better this way, since `selectors` isn't meant to be exposed to the wrapped component - can you add repeat this change elsewhere?